DOC 2126: Writing a CorDapp using a template tutorials rewrite (#483)
corda/corda-docs: content/en/docs/corda-os/4.8/tut-two-party-introduction.md

# Applying contract constraints

> In the previous steps of the [Writing a CorDapp using a template](writing-a-cordapp-using-a-template) tutorial, you built a CorDapp which enabled you to model IOUs on ledger.

Reviewer: Minor note on consistency - Line 29 refers to 'on the blockchain', but here we talk about 'on ledger'.

Author: It should be "on ledger".

> In the previous steps of the [Writing a CorDapp using a template](writing-a-cordapp-using-a-template) tutorial, you built a CorDapp which enabled you to model IOUs on ledger.

Reviewer 2: Tiny, but there's an opportunity for simpler language here:

```suggestion
Previously in the  [Writing a CorDapp using a template](writing-a-cordapp-using-a-template) tutorial, you built a CorDapp which let you model IOUs on ledger.
```
